hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/ScanOptions.java

```diff
@@ -64,4 +66,10 @@ default void readAllVersions() {
   void setKeepDeletedCells(KeepDeletedCells keepDeletedCells);
 
   KeepDeletedCells getKeepDeletedCells();
+
+  int getMinVersions();
+
+  void setMinVersions(int minVersions);
+
+  Scan getScan() throws IOException;
```

Comment:
Please add a comment to say that the returned Scan object is only for reading, changing it will have no effect to the actual scan operation.
And please modify the javadoc of this class to indicate that now we can also change min versions?

Reply:
Done